### PR TITLE
Update documentation regarding containerlab migration.

### LINKED ISSUE #77: Update Documentation
Update documentation for containerlab based mini-lab.

### PR DESCRIPTION
**Pull request by Gerrit91**
Closes #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
...
+sudo chmod +x /usr/local/bin/kind
 
 # Install docker-compose, for more details see https://docs.docker.com/compose/install/
-sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo curl -Lo /usr/local/bin/docker-compose "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)"
 sudo chmod +x /usr/local/bin/docker-compose
 ```
 
-The following ports are getting used statically:
+The following ports are used statically on your host machine:
```
Comment (GrigoriyMikhalkin) [inline README.md:58]: Do we still use all these ports?

File: README.md [hunk 2]
```diff
@@ -1,70 +1,61 @@
 # mini-lab
 
-The mini-lab is a small, virtual setup to locally run the metal-stack. It deploys the metal control plane and a partition with two simulated leaf switches. The lab can be used for trying out metal-stack, demonstration purposes or development.
+The mini-lab is a small, virtual setup to locally run the metal-stack. It deploys the metal control plane and a metal-stack partition with two simulated leaf switches. The lab can be used for trying out metal-stack, demonstration purposes or development.
 
 ![overview components](docs/overview.png)
```
Comment (GrigoriyMikhalkin) [inline README.md:5]: Should we redraw it completely, similar to how `containerlab` presents topologies in it's docs: https://containerlab.srlinux.dev/lab-examples/cfg-clos/#description, but with different(metal-stack like) palette?

File: README.md (hunk 1 above)
Comment (Gerrit91) [reply README.md:58]: Yes... but maybe we can get rid off it with the proposal I made here: https://github.com/metal-stack/mini-lab/issues/80#issuecomment-968623041

File: README.md (hunk 2 above)
Comment (Gerrit91) [reply README.md:5]: That'd be great. But it's also quite a lot of work. :sweat:

Comment (GrigoriyMikhalkin) [reply README.md:5]: I will try to do some proto. But there's big chance it will look quirky)

**Review by majst01 — APPROVED** [review]

**Pull request merged**